component-playground/config.js

```diff
@@ -24,10 +26,15 @@ try {
   }
 }
 
-var webpackConfig = {
+config.server = _.extend({
+  port: 8989,
+  hostname: 'localhost'
+}, userConfig.server);
+
+config.webpack = {
   context: playgroundPath,
   entry: [
-    'webpack-dev-server/client?http://localhost:8989',
+    'webpack-dev-server/client?http://localhost:' + config.server.port,
```

Comment:
You also need to replace localhost with `' + config.server.hostname + '`.

Please test this yourself as well afterwards.

PS. It's going to be > 80 chars so something like this would do:

```js
    'webpack-dev-server/client?http://' + config.server.hostname + ':' + 
                                          config.server.port,
```